Link to `hap CLI tool readme` in hap documentation (#1922)
kyma-project/kyma-environment-broker: docs/contributor/03-11-hap-rules.md

  - free                                # hyperscalerType=aws, !dirty
                                        # hyperscalerType=azure, !dirty
```

## CLI Tool

> There is CLI tool available for validation of rules. To get more details on building and using this tool, see [`README`](../../cmd/parser/README.md).

Reviewer: ```suggestion
A CLI tool for validating rules is available. For more details on building and using this tool, see [HAP Parser](../../cmd/parser/README.md).
```
